src/api/pool_api.js

```diff
@@ -552,6 +565,13 @@ module.exports = {
                 identity: {
                     $ref: 'common_api#/definitions/access_key'
                 },
+                fs_path: {
+                    type: 'string'
+                },
+                fs_backend: {
+                    type: 'string',
+                    enum: ['CEPH_FS', 'GPFS']
+                },
```

Comment:
why is this flat here? perhaps would be best to move nsfs_config to the definitions and $ref to it from all those places we use it?

Reply:
it's flat because I wanted to keep the same behavior as for the connection properties (endpoint, endpoint_type etc are flat here too).